tiflash/monitor-tiflash.md

```diff
@@ -8,30 +8,71 @@ aliases: ['/docs/dev/tiflash/monitor-tiflash/','/docs/dev/reference/tiflash/moni
-| `tiflash_coprocessor_request_handle_seconds` | The histogram of the processing time of each coprocessor request, in which the processing time is from starting to execute the coprocessor request to completing the execution. `batch` is the processing time of batch request. `cop` is the processing time of coprocessor requests that are sent directly via the coprocessor interface. |
-| `tiflash_coprocessor_response_bytes` | The total bytes of the response. |
-
-## Monitor DDL operations
-
-| Monitoring items | Description |
-|:---|:-----|
-| `tiflash_schema_version` | The version of the schema currently cached in TiFlash. |
-| `tiflash_schema_apply_count` | This item includes the count of three types of `appy`: `diff apply`, `full apply`, and `failed apply`. `diff apply` is the normal process of a single apply. If `diff apply` fails, `failed apply` increases by `1`, and TiFlash rolls back to `full apply`. |
-| `tiflash_schema_internal_ddl_count` | The number of specific DDL operations in TiFlash. |
-| `tiflash_schema_apply_duration_seconds` | The time used for a single `apply schema` operation. |
-
-## Monitor Raft
-
-| Monitoring items | Description |
-|:---|:-----|
-| `tiflash_raft_read_index_count` | The number of times that the coprocessor triggers the `read_index` request, which equals to the number of Regions triggered by a coprocessor. |
-| `tiflash_raft_read_index_duration_seconds` | The time used by `read_index`. Most time is used for interaction with Leader and retry. |
-| `tiflash_raft_wait_index_duration_seconds` | The time used by `wait_index`, namely the time used to wait until local index >= read_index after the `read_index` request is received. |
+If you use TiUP or TiDB Ansible to deploy the TiDB cluster, the monitoring system (Prometheus & Grafana) is deployed at the same time. For more information, see [Overview of the Monitoring Framework](/tidb-monitoring-framework.md).
+
+The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Disk Performance, and so on. A lot of metrics are there to help you diagnose.
+
+TiFlash has three dashboard panels: **TiFlash-Summary**, **TiFlash-Proxy-Summary**, and **TiFlash-Proxy-Details**. The metrics on these panels indicate the current status of TiFlash. The **TiFlash-Proxy-Summary** and **TiFlash-Proxy-Details** panel mainly shows the information of the Raft layer and the metrics are detailed in [Key Monitoring Metrics of TiKV](/grafana-tikv-dashboard.md).
+
+> **Note:**
+>
+> It is recommended that you use TiDB v4.0.5 or later versions for improved monitor on TiFlash.
```

Comment:
4.0.5 release is scheduled on Aug 28. Can we change the version number to v4.0.4 for now?

Reply:
@JaySon-Huang PTAL

Reply:
We have added/changed many panels in v4.0.5. I recommend to keep it as v4.0.5.